.gitignore

```diff
@@ -1 +1,5 @@
-gen/
\ No newline at end of file
+gen/
+*.name
```

Comment:
Revert all changes to this file; no changes should be included in this pull request.